mobile/.env

```
@@ -0,0 +1 @@
+EXPO_PUBLIC_API_URL=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)